doc/nrf/images/bt_mesh_model_callbacks.svg

```diff
@@ -0,0 +1,259 @@
+		<g id="shape1030-135" transform="translate(110.67,-512.854)">
+			<title>Nordic Lake.1030</title>
+			<desc>Settings subsystem</desc>
+			<rect x="0" y="513.854" width="87.1738" height="34.0157" class="st13"/>
+			<text x="24.12" y="527.26" class="st14">Settings <tspan x="17.74" dy="1.2em" class="st15">subsystem</tspan></text>		</g>
+		<g id="shape1031-139" transform="translate(219.979,-512.854)">
+			<title>Nordic Lake.1031</title>
+			<desc>Mesh</desc>
+			<rect x="0" y="513.854" width="87.1738" height="34.0157" class="st13"/>
+			<text x="29.97" y="534.46" class="st14">Mesh</text>		</g>
+		<g id="shape1032-142" transform="translate(329.288,-512.854)">
+			<title>Nordic Lake.1032</title>
+			<desc>Access</desc>
+			<rect x="0" y="513.854" width="87.1738" height="34.0157" class="st13"/>
+			<text x="27.36" y="534.46" class="st14">Access</text>		</g>
+		<g id="shape1033-145" transform="translate(438.597,-512.854)">
+			<title>Nordic Lake.1033</title>
+			<desc>Model</desc>
+			<rect x="0" y="513.854" width="87.1738" height="34.0157" class="st13"/>
+			<text x="27.78" y="534.46" class="st14">Model</text>		</g>
+		<g id="shape1106-148" transform="translate(1.3008,-12.0594)">
+			<title>Nordic Sky.1074</title>
+			<desc>Mesh device is unprovisioned.</desc>
+			<rect x="0" y="529.811" width="524.842" height="18.0594" class="st11"/>
+			<text x="201.05" y="541.84" class="st12">Mesh device is unprovisioned.</text>		</g>
+		<g id="shape1107-151" transform="translate(391.171,-403.766)">
+			<title>Nordic Lake.1107</title>
+			<desc>The Access API is not ready for use yet. The model may do add...</desc>
+			<rect x="0" y="501.098" width="134.671" height="46.7717" class="st16"/>
+			<text x="5.28" y="514.28" class="st12">The Access API is not ready for <tspan x="12.56" dy="1.2em" class="st15">use yet. The model may do </tspan><tspan
+						x="9.15" dy="1.38em" class="st15">additional initialization here.</tspan></text>		</g>
+		<g id="shape1108-156" transform="translate(399.533,-256.668)">
+			<title>Nordic Lake.1108</title>
+			<desc>The model restores its state from the persistent storage.</desc>
+			<rect x="0" y="513.854" width="126.309" height="34.0157" class="st16"/>
+			<text x="6.54" y="526.66" class="st12">The model restores its state <tspan x="6.18" dy="1.38em" class="st15">from the persistent storage.</tspan></text>		</g>
+		<g id="shape1109-160" transform="translate(379.85,-188.536)">
+			<title>Nordic Lake.1109</title>
+			<desc>All Access APIs are ready for use. The model may start its be...</desc>
+			<rect x="0" y="513.854" width="145.991" height="34.0157" class="st16"/>
+			<text x="6.43" y="526.66" class="st12">All Access APIs are ready for use. <tspan x="4.84" dy="1.38em" class="st15">The model may start its behavior.</tspan></text>		</g>
+		<g id="shape1110-164" transform="translate(399.533,-63.6088)">
+			<title>Nordic Lake.1110</title>
+			<desc>The model resets its state from the persistent storage.</desc>
```

Comment:
Suggestion: "Model resets its state and clears stored data (if any) from persistent storage"

Reply:
It needs to be erased manually, in the sense, an implementer should write corresponding code as shown on the last arrow of the diagram.